addons/html_builder/static/src/core/builder_content_editable_plugin.js

```diff
@@ -1,34 +1,55 @@
 import { Plugin } from "@html_editor/plugin";
+import { selectElements } from "@html_editor/utils/dom_traversal";
 import { registry } from "@web/core/registry";
 
 export class BuilderContentEditablePlugin extends Plugin {
     static id = "builderContentEditablePlugin";
     resources = {
-        force_not_editable_selector: [
+        content_not_editable_selector: [
```

Comment:
Resources names should be plural because you're pushing them in an array. If you do `getResource("content_not_editable_selector")` you would expect to have a `string` as return value, but it's not the case because when you do `getResource("those_things")` it is always an array.

Same for `content_editable_selector`, obviously.